src/cogs/threads_help_tickets.py

```diff
@@ -1,36 +1,132 @@
 from __future__ import annotations
 
 import asyncio
+import re
 from datetime import datetime, timedelta, timezone
 from functools import partial
-from typing import TYPE_CHECKING, cast
+from typing import TYPE_CHECKING, NamedTuple, cast
 
 import discord
 from discord import app_commands, ui
 from discord.ext import commands
-from discord.utils import find
+from discord.utils import find, get
 from typing_extensions import Self
 
-from utils.constants import ASK_CHANNEL_ID, ASK_MESSAGE_ID, BUG_CENTER_ID
+from utils.constants import BUG_CENTER_ID, NEW_REQUEST_CHANNEL_ID, REQUEST_MESSAGE_ID, REQUESTS_CHANNEL_ID
+from utils.types import Snowflake
 
 if TYPE_CHECKING:
+    from discord import RawThreadDeleteEvent, RawThreadUpdateEvent
+
     from main import HelpCenterBot
 
+THREAD_ID_FROM_CONTENT = re.compile(r"\[.+\]\(https:\/\/discord\.com\/channels\/\d{18,20}\/(\d{18,20})\)")
```

Comment:
This constant seems to only be used once in the code, it might be better to simply passing the value when it is used instead of declaring a global-scoped variable